pages/Documents/MessagingChannels/ProactiveMessaging/api.md

```diff
@@ -24,6 +24,18 @@ Note: Proactive Messaging can be leveraged using Proactive 2.0 API or the [Web T
     * Click on Enable button to onboard to proactive api.
     * Click on show secrets to get app Id and secrets which will be used to create APP JWT. Click here to learn how to use [APP JWT](https://developers.liveperson.com/connector-api-send-api-authorization-and-authentication.html#get-appjwt).
 
+### Proactive 2.0 Product Capabilities
+
+| Details | 2.0 API |
+| :--- | :--- |
+| Channel | SMS <br /> WhatsApp <br /> In-App |
+| Rate throttling | Upto 10 requests per second |
+| Send rate |  SMS - 9** MPS <br /> WhatsApp - 9** MPS <br /> In-App - 9** MPS <br /> <br /> MPS - message per sec |
```

Comment:
This can be configured.  Maybe confirm with Kuntal if we want to mention this or not

Reply:
Yeah mentioned it below, to update contact proactive team